app/src/main/java/org/mifos/mobilebanking/ui/fragments/BeneficiaryListFragment.java

```diff
@@ -94,8 +94,10 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
     @Override
     public void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
-        outState.putParcelableArrayList(Constants.BENEFICIARY, new ArrayList<Parcelable>(
-                beneficiaryList));
+        if (beneficiaryList != null) {
```

Comment:
@therajanmaurya Without null pointer check, the app was crashing. (Maybe because of migration from Fragment to activity)